test/functional/logging_test.rb

```diff
@@ -1,27 +1,25 @@
 # Logging and deprecation facilities functional tests
 
 require 'functional/helper'
-require 'byebug'
 
 describe 'Deprecation Facility Behavior' do
   include FunctionalHelper
 
   let(:profile) { File.join(profile_path, 'deprecation', profile_name) }
   let(:invocation) { "exec #{profile} #{control_flag}" }
   # Running in JSON mode has the side-effect of sending log messages to STDERR
-  let(:run_result) { run_inspec_process(invocation, json: true)}
+  let(:run_result) { run_inspec_process(invocation, json: true) }
 
   # Expect one control, 3 results
   let(:json_result) { run_result.payload.json['profiles'][0]['controls'][0]['results'] }
 
   describe 'when the deprecation is in a custom resource and the deprecate DSL method is used' do
     describe 'when the action is to fail the control' do
-
       describe 'when the resource is called in a control' do
         let(:profile_name) { 'typical' }
         let(:control_flag) { '--controls deprecate_fail_mode' }
 
-        it "should result in a failed control" do
+        it 'should result in a failed control' do
```

Comment:
I think chefstyle will want us to change these all to `"`, so might want to leave these for now, but I agree on principal =)

Reply:
Yup, I am all for one OR the other, but must be the same in a single file.

Reply:
In that case they should just all be `"` because with `'` we'll still need to change to `"` for anything that does interpolation (unless you use a percent literal like `%Q[]`, which I don't think we do often/anywhere)

Reply:
I agree, but this breaks our RuboCop currently (even though `test/` isn't scanned).

Reply:
Ok, lets just note that then and we'll churn this after chefstyle =)